tools/install_envs.sh

```diff
@@ -0,0 +1,12 @@
+# script must be executed from root git directory
+pip install -r requirements.txt
+pip install -r test_requirements.txt
+pip install -r src/server/server/requirements.txt
```

Comment:
Those requirements are installed in "default" environment and in appengine_env virtual env, should it not be installed only on appengine_env virtual env ?